backend-services/src/data_service/operations/bookmark.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+from motor.motor_asyncio import AsyncIOMotorClient, AsyncIOMotorCollection
+
+from data_service.schema.actions import (
+    CreateBookmark,
+    CreateBookmarkResult,
+    DeleteBookmark,
+    DeleteBookmarkResult,
+    GetBookmarks,
+    GetBookmarksResult,
+)
+from data_service.schema.entities import Bookmark, BookmarkDocument
+from data_service.settings import MongoSettings
+
+mongo_settings = MongoSettings()
```

Comment:
I think these should be all caps, e.g. MONGO_SETTINGS, does the linter not complain?

Reply:
The value of `mongo_settings` depends on information known only at runtime and is therefore not constant.  All caps are reserved to values that, essentially, are known at the time of writing the code (compile time if you prefer).